seed/views/v3/geocode.py

```diff
@@ -0,0 +1,145 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+from django.db.models import Subquery
+from django.http import JsonResponse
+
+from drf_yasg.utils import swagger_auto_schema
+
+from rest_framework import viewsets
+from rest_framework.decorators import action
+
+from seed.decorators import ajax_request_class
+
+from seed.lib.superperms.orgs.decorators import has_perm_class
+
+from seed.models.properties import PropertyState, PropertyView
+from seed.models.tax_lots import TaxLotState, TaxLotView
+from seed.utils.api import api_endpoint_class, OrgMixin
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.geocode import geocode_buildings
+
+
+class GeocodeViewSet(viewsets.ViewSet, OrgMixin):
+
+    @swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field()],
+        request_body=AutoSchemaHelper.schema_factory(
+            {
+                'property_view_ids': ['integer'],
+                'taxlot_view_ids': ['integer'],
+            },
+            description='IDs by inventory type for records to be geocoded.'
+        )
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('can_modify_data')
+    @action(detail=False, methods=['POST'])
+    def geocode_by_ids(self, request):
```

Comment:
Could we get a short docstring for the view here? Same for the other views added in this PR

Reply:
Nice catch! Added them in, let me know if you had something else in mind.